libs/platform/src/lib/components/table/table.component.scss

```diff
@@ -0,0 +1,416 @@
+$fd-table-border: 0.0625rem solid #e4e4e4;
```

Comment:
I believe this file was added accidentally

Reply:
@mikerodonnell89 why?

Reply:
It is in a directory by itself
<img width="357" alt="Screen Shot 2021-12-16 at 9 40 24 AM" src="https://user-images.githubusercontent.com/2471874/146392214-9f01617d-0c3a-4738-b086-270ff6339c1e.png">

And the first few lines are essentially a duplicate of [this scss file](https://github.com/SAP/fundamental-ngx/blob/0a69acfa9595e168458db82e5f605c88994cc634/libs/platform/src/lib/table/table.component.scss). Note the almost identical pathnames except this file one contains the directory "components".  Should the changes here actually be in [this file](https://github.com/SAP/fundamental-ngx/blob/0a69acfa9595e168458db82e5f605c88994cc634/libs/platform/src/lib/table/table.component.scss) or is it intentional to have these styles in their own directory?
 
Same question for the constants/index.ts file above